CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+## Version 4.3.0
+
+- Leverage Pydantic validation for enum choices, enabling more complex use-cases
+
```

Comment:
Can you also bump the version in `_version.py` to `4.3.0`?

https://github.com/mpkocher/pydantic-cli/blob/master/pydantic_cli/_version.py

Reply:
Done